sources/packages/web/src/views/aest/institution/ApplicationOfferingChangeRequestForm.vue

```diff
@@ -0,0 +1,167 @@
+        />
+      </v-window-item>
+    </v-window>
+    <assess-offering-change-modal ref="assessOfferingChangeModal" />
+  </full-page-container>
+</template>
+
+<script lang="ts">
+import { ref, defineComponent, computed, onMounted } from "vue";
+import { RouteLocationRaw } from "vue-router";
+import {
+  ApplicationOfferingChangeRequestHeader,
+  ApplicationOfferingChangeRequestStatus,
+} from "@/types";
+import { ApplicationOfferingDetailsReviewAPIOutDTO } from "@/services/http/dto";
+import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
+import ApplicationOfferingChangeDetailsHeader from "@/components/aest/institution/ApplicationOfferingChangeDetailsHeader.vue";
+import { ApplicationOfferingChangeRequestService } from "@/services/ApplicationOfferingChangeRequestService";
+import StudentApplicationOfferingChangeDetails from "@/components/aest/StudentApplicationOfferingChangeDetails.vue";
+import OfferingForm from "@/components/common/OfferingForm.vue";
+import { BannerTypes } from "@/types/contracts/Banner";
+import { ApplicationOfferingDetails } from "@/types/contracts/StudentApplicationOfferingChangeContract";
+
+export default defineComponent({
+  components: {
+    StudentApplicationOfferingChangeDetails,
+    ApplicationOfferingChangeDetailsHeader,
+    OfferingForm,
+  },
+  props: {
+    studentId: {
+      type: Number,
+      required: true,
+    },
+    applicationId: {
+      type: Number,
+      required: true,
+    },
+    applicationOfferingChangeRequestId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup(props) {
+    const tab = ref("requested-change");
+    const headerDetails = ref({} as ApplicationOfferingChangeRequestHeader);
+    const applicationOfferingChangeRequestDetails = ref(
+      {} as ApplicationOfferingDetailsReviewAPIOutDTO,
+    );
+    const studentApplicationOfferingDetails = ref(
+      {} as ApplicationOfferingDetails,
+    );
+    onMounted(async () => {
+      applicationOfferingChangeRequestDetails.value =
+        await ApplicationOfferingChangeRequestService.shared.getApplicationOfferingDetailsForReview(
+          props.applicationOfferingChangeRequestId,
+        );
+      studentApplicationOfferingDetails.value = {
+        studentName:
+          applicationOfferingChangeRequestDetails.value.studentFullName,
+        applicationNumber:
+          applicationOfferingChangeRequestDetails.value.applicationNumber,
+        locationName:
+          applicationOfferingChangeRequestDetails.value.locationName,
+        reasonForChange: applicationOfferingChangeRequestDetails.value.reason,
+        accessedNoteDescription:
+          applicationOfferingChangeRequestDetails.value.assessedNoteDescription,
+        applicationOfferingChangeRequestStatus:
+          applicationOfferingChangeRequestDetails.value.status,
+      };
+      headerDetails.value = {
+        submittedDate:
+          applicationOfferingChangeRequestDetails.value.submittedDate?.toString(),
```

Comment:
Minor but is there a reason to do a `toString` at lines 130 137 and 142?
The values can be kept as they are since they will be converted either way by `dateOnlyLongString`, right?

Reply:
I have changed it to use `dateOnlyLongString`.